maintenance_plan/models/maintenance_equipment.py

```diff
@@ -80,6 +81,9 @@ def _prepare_request_from_plan(self, maintenance_plan, next_maintenance_date):
             "note": maintenance_plan.note,
             "company_id": maintenance_plan.company_id.id or self.company_id.id,
         }
+        if "planned_hours" in request_model._fields:
```

Comment:
You always forget to comment why doing these things that will serve in the future. In this case, a comment saying that this field comes from `maintenance_timesheet` and for avoiding a glue module just for that.